### PR TITLE
Add forward endpoint

### PR DESCRIPTION
**Pull request by LovisaLugnegard**

### CODE REVIEW

File: internal/http/services/ocmd/invites.go [hunk 1]
```diff
@@ -103,53 +100,40 @@ func (h *invitesHandler) generateInviteToken(w http.ResponseWriter, r *http.Requ
...
-		return
-	}
-	var req Request
-
-	// Try to decode the request body into the struct. If there is an error,
-	// respond to the client with the error message and a 400 status code.
-	err := json.NewDecoder(r.Body).Decode(&req)
-	if err != nil {
-		http.Error(w, err.Error(), http.StatusBadRequest)
+	if r.FormValue("token") == "" || r.FormValue("domain") == "" {
```
Comment (ishank011) [inline internal/http/services/ocmd/invites.go]: Instead of domain, can wel call it provider?

File: internal/http/services/ocmd/invites.go [hunk 2]
```diff
@@ -103,53 +100,40 @@ func (h *invitesHandler) generateInviteToken(w http.ResponseWriter, r *http.Requ
...
-		Token:  "blbl",
-		UserId: contextUser.GetId(),
-		Expiration: &types.Timestamp{
-			Nanos:   uint32(expireTime.UnixNano()),
-			Seconds: uint64(expireTime.Unix()),
-		},
+		Token: r.FormValue("token"),
+	}
+
+	domainInfo, err := gatewayClient.GetInfoByDomain(ctx, &ocmprovider.GetInfoByDomainRequest{
```
Comment (ishank011) [inline internal/http/services/ocmd/invites.go]: Same. providerInfo instead of domainInfo

File: internal/http/services/ocmd/invites.go [hunk 3]
```diff
@@ -103,53 +100,40 @@ func (h *invitesHandler) generateInviteToken(w http.ResponseWriter, r *http.Requ
...
 
-	//TODO Update these values with values from GetInfoByDomain response
 	forwardInviteReq := &invitepb.ForwardInviteRequest{
 		InviteToken: token,
 		OriginSystemProvider: &ocmprovider.ProviderInfo{
-			Domain:         "domain",
-			ApiVersion:     "ApiVersion",
-			ApiEndpoint:    "APIEndPoint",
-			WebdavEndpoint: "WebdavEndpoint",
+			Domain:         r.FormValue("domain"),
```
Comment (ishank011) [inline internal/http/services/ocmd/invites.go]: Just pass the object you got from GetInfoByDomain instead of constructing it again

**Review by ishank011 — CHANGES_REQUESTED** [review]

File: internal/http/services/ocmd/invites.go (hunk 1 above)
Comment (LovisaLugnegard) [reply internal/http/services/ocmd/invites.go]: Sure, do you thunk providerDomain makes even more sense?

Comment (ishank011) [reply internal/http/services/ocmd/invites.go]: Yeah cool

**Pull request merged**